### PR TITLE
Completed Italian Translation (Incl. Nerubar Palace + Khazar Algar)

### PR DESCRIPTION
**Pull request by VassalloAlby**
Completed the IT translation
Some terms were left in english as the italian community uses them frequently in their wow speech

### CODE REVIEW

File: Core/Locales/common.itIT.lua [hunk 1]
```diff
@@ -130,8 +130,8 @@ L.link_with_rticon = "{rt%d}Vincolato a %s"
 L.link_both = "%s vincolato a %s" -- XXX needs updated
 L.link_both_icon = "|T13700%d:0|t%s vincolato a |T13700%d:0|t%s" -- XXX needs updated
```
Comment (funkydude) [inline Core/Locales/common.itIT.lua:131]: Please update both of these as the English version has changed.

File: Core/Locales/common.itIT.lua [hunk 2]
```diff
@@ -120,7 +120,7 @@ L.affixes = "Modificatori" -- Seasonal affixes for raids and mythic+ dungeons
 L.marker = "Icona bersaglio %s"
 L.marker_player_desc = "Marchia i giocatori affetti da %s con %s, necessario ruolo assistente o capo." -- Mark players affected by 'SPELL_NAME' with SKULL_ICON
 L.marker_npc_desc = "Marchia %s con %s, necessario ruolo assistente o capo." -- Mark NPC_NAME with SKULL_ICON
---L.marker_npc_aura_desc = "Mark NPCs affected by '%s' with %s, requires promoted or leader." -- Mark NPCs affected by 'SPELL_NAME' with SKULL_ICON
+--L.marker_npc_aura_desc = "Marchia NPCs che hanno '%s' con %s, richiede promozione or capogruppo." -- Mark NPCs affected by 'SPELL_NAME' with SKULL_ICON
```
Comment (funkydude) [inline Core/Locales/common.itIT.lua:123]: You forgot to remove the `--` comment.

**Comment by funkydude** [comment]
`--L.tempNPMsg = "Our season 1 nameplate timers are active."`
Missed this one.

File: Locales/itIT.lua [hunk 3]
```diff
@@ -85,21 +85,21 @@ L.spell_under_you = "BigWigs: Abilità sotto di te"
...
+L.compartmentMenu = "No Icona menu addon"
+L.compartmentMenu_desc = "Disattivando quest'opzione farà in modo che BigWigs figurerà nella zona addon del menu. Consigliamo di lasciare quest'opzione abilitata."
 L.configure = "Configura"
 L.resetPositions = "Ripristina le Posizioni"
 L.colors = "Colori"
 L.selectEncounter = "Seleziona il Combattimento"
---L.privateAuraSounds = "Private Aura Sounds"
---L.privateAuraSounds_desc = "Private auras can't be tracked normally, but you can set a sound to be played when you are targeted with the ability."
+L.privateAuraSounds = "Suoni per le Aura Private"
+L.privateAuraSounds_desc = "Le Aure Private non possono essere visualizzate normalmente, ma puoi impostare un suono che suonerà quando sei il bersaglio dell'abilità"
```
Comment (funkydude) [inline Locales/itIT.lua:102]: Missed a full stop here.

File: Locales/itIT.lua [hunk 4]
```diff
@@ -178,82 +178,81 @@ L.tankhealer = "|cFFFF0000Messaggio per Difensori e Guaritori.|r "
...
+L.export = "Esporta"
+L.export_info = "Seleziona quali impostazioni vorresti esportare e condividere con gli altri.\n\n|cffff4411 Puoi solo condividere impostazioni generali e quelle non hanno nessun effetto sulle impostazioni specifiche ai boss.|r"
+L.export_string = "Esporta Stringa"
+L.export_string_desc = "Copia questa stringa BigWigs se vuoi condividere le tue impostazioni."
+L.import_string = "Improta Stringa"
+L.import_string_desc = "Incolla la stringa BigWigs che vuoi importare qua."
+L.position = "Posizione"
+L.settings = "Impostazioni"
+L.other_settings = "Altre Impostazioni"
+L.nameplate_settings_import_desc = "Importa tutte le impostazioni delle barre delle unità"
```
Comment (funkydude) [inline Locales/itIT.lua:194]: Missed a full stop.

File: Locales/itIT.lua [hunk 5]
```diff
@@ -178,82 +178,81 @@ L.tankhealer = "|cFFFF0000Messaggio per Difensori e Guaritori.|r "
...
+L.position_export_bars_desc = "Esporta la posizone (ancoraggi) delle barre."
+L.position_export_messages_desc = "Esporta la posizone (ancoraggi) dei messaggi."
+L.position_export_countdown_desc = "Esporta la posizone (ancoraggi) del conto alla rovescia."
+L.settings_import_bars_desc = "Importa le impostazioni generali delle barre come dimensioni, font, ecc."
+L.settings_import_messages_desc = "Importa le impostazioni generali dei messaggi come dimensioni, font, ecc."
+L.settings_import_countdown_desc = "Importa le impostazioni generali del conto alla rovescia come dimensioni, font, ecc."
+L.settings_export_bars_desc = "Esporta le impostazioni generali delle barre come dimensioni, font, ecc."
+L.settings_export_messages_desc = "Esporta le impostazioni generali dei messaggi come dimensioni, font, ecc."
+L.settings_export_countdown_desc = "Esporta le impostazioni generali del conto alla rovescia come dimensioni, font, ecc."
+L.colors_import_bars_desc = "Importa i colori delle barre"
```
Comment (funkydude) [inline Locales/itIT.lua:208]: Missed a full stop.

File: Locales/itIT.lua [hunk 6]
```diff
@@ -178,82 +178,81 @@ L.tankhealer = "|cFFFF0000Messaggio per Difensori e Guaritori.|r "
...
---L.level1 = "Level 1"
---L.level2 = "Level 2"
---L.level3 = "Level 3"
---L.N10 = "Normal 10"
---L.N25 = "Normal 25"
---L.H10 = "Heroic 10"
---L.H25 = "Heroic 25"
+L.timewalk = "Viaggi nel tempo"
+L.story = "Storia"
+L.mplus = "Mythic+ %d"
```
Comment (funkydude) [inline Locales/itIT.lua:249]: Should this not be `Mitica+`?

File: Plugins/Locales/itIT.lua [hunk 7]
```diff
@@ -190,7 +190,7 @@ L.blockTalkingHeadTimewalking = "Viaggi nel Tempo (Spedizioni & Incursioni)"
 L.blockTalkingHeadScenarios = "Scenari"
 
 L.redirectPopups = "Reindirizza i banner popup nelle su bigwigs"
---L.redirectPopupsDesc = "Popup banners in the middle of your screen such as the '|cFF33FF99vault slot unlocked|r' banner will instead be displayed as BigWigs messages. Questi banner possono essere abbastanza grandi, durare molto, e bloccare la possibilità di cliccarci attraverso."
+L.redirectPopupsDesc = "I banner popup in mezzo al tuo schermo per esempio '|cFF33FF99vault slot unlocked|r' verrano invece visualizzati come messaggi BigWigs. Questi banner possono essere abbastanza grandi, durare molto, e bloccare la possibilità di cliccarci attraverso."
```
Comment (funkydude) [inline Plugins/Locales/itIT.lua:193]: You missed the `vault slot unlocked` part, it should reflect the translation of the message you see in game.

File: Plugins/Locales/itIT.lua [hunk 8]
```diff
@@ -302,76 +302,76 @@ L.fadeTimeDesc = "Dopo quanti secondi il messaggio deve scomparire, in secondi"
 -- Nameplates.lua
 --
 
---L.nameplates = "Nameplates"
---L.testNameplateIconBtn = "Show Test Icon"
---L.testNameplateIconBtn_desc = "Creates a test icon for you to test your current display settings with on your targeted nameplate."
```
Comment (funkydude) [inline Plugins/Locales/itIT.lua:307]: You removed the full stop from nearly every single "desc" (description) translation here.

**Comment by funkydude** [comment]
Thanks for your work! Added a few comments that need fixed.

File: Core/Locales/common.itIT.lua (hunk 1 above)
Comment (VassalloAlby) [reply Core/Locales/common.itIT.lua:131]: Hi, where can I find the english version to use as a reference ?

**Comment by VassalloAlby** [comment]
Thanks for the comments, should have updated everything you mentioned. I used the en-US locale as a reference for the ones you marked as to be updated

**Comment by funkydude** [comment]
Thanks!

**Pull request merged**